zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -35,6 +35,12 @@ def set_error(self, error):
         self.error = error
 
 
+class PingRsp(AgentResponse):
+    def __init__(self):
+        super(CopyRsp, self).__init__()
```

Pylint (3.3.7)

[error] Bad first argument 'CopyRsp' given to super()

(E1003)